axelrod/tests/unit/test_tournament.py

```diff
@@ -154,20 +308,24 @@ def test_no_progress_bar_play(self):
 
 
```

Comment:
That looks fine to me and your name is fine (perhaps the slightly shorted version below).

When called can we be verbose:

`assert_play_pbar_correct_total_and_finished(pbar=pbar, total=10)`